elastic/kibana
# [Uptime] Right-align IP address column in pings list

## Summary

As part of our [redesign work](https://github.com/elastic/kibana/issues/40583) this column should've been right-aligned. This change updates that column.

## Testing
0. Start up Kibana with several uptime monitors
1. Navigate to the _Monitor_ page for one of the monitors, observe the column in the table

### Checklist

Use ~~strikethroughs~~ to remove checklist items you don't feel are applicable to this PR.


### For maintainers
